opencti-platform/opencti-graphql/src/utils/confidence-level.ts

```diff
@@ -14,6 +15,15 @@ type ObjectWithConfidence = {
 };
 
 export const computeUserEffectiveConfidenceLevel = (user: AuthUser) => {
+  // if a user has BYPASS capability, we consider a level 100
+  if (isUserHasCapability(user, BYPASS)) {
```

Comment:
I suggest to use `isBypassUser(user)` instead of `isUserHasCapability(user, BYPASS)`, since in `isUserHasCapability` we first test `isBypassUser`.